rps.js

```diff
@@ -0,0 +1,15 @@
+const Playgame = function(userpicked) {
+
+    let random = Math.random();
+    if (random <= 0.33)
```

Comment:
write if functions similar to this this will make the code easier to read and with less errors

`if (random <= 0.33) {
computerpick = 'Rock'
} else if (random > 0.33 && random <= 0.66 ) {
computerpick = 'Paper'
} else if (random 0.66) {
computerpick = 'Scissors'
}`